exchange/seat_non_bids.go

```diff
@@ -53,3 +58,55 @@ func (snb *nonBids) get() []openrtb_ext.SeatNonBid {
 	}
 	return seatNonBid
 }
+
+// newProxyNonBid creates a new proxy non-bid object with the given impression ID and non-bid reason.
+func newProxyNonBid(impId string, nonBidReason int) openrtb_ext.NonBid {
+	return openrtb_ext.NonBid{
+		ImpId:      impId,
+		StatusCode: nonBidReason,
+	}
+}
```

Comment:
This function doesn't provide much functionality IMHO and introduces a new term "proxy non bid" which I don't understand. Callers can use: 

```
openrtb_ext.NonBid{ImpId: impId, StatusCode: nonBidReason}
```

instead of 

```
newProxyNonBid(impId, nonBidReason)
```

Reply:
@SyntaxNode: Renaming it to `newNonBid` instead